pennylane/tape/__init__.py

```diff
@@ -23,7 +23,7 @@
 from .queuing import AnnotatedQueue, Queue, QueuingContext
 from .measure import MeasurementProcess, state
 from .qnode import QNode, qnode
-from .tapes import QuantumTape, QubitParamShiftTape, CVParamShiftTape, ReversibleTape
+from .tapes import BaseQuantumTape, QuantumTape, QubitParamShiftTape, CVParamShiftTape, ReversibleTape

```

Comment:
minor, but wondering if it makes more sense to have `QuantumTape` and `JacobianTape`? That way it is easier to tell the difference (e.g., the latter adds jacobian functionality, whereas the former is simply a tape recorder of quantum operations)

Reply:
Oh yes, good idea!